integration-tests/environments/react-native-test-app/ios/Podfile

```diff
@@ -2,6 +2,8 @@ require_relative '../../../../node_modules/react-native-test-app/test_app'
 
 workspace 'RealmTests.xcworkspace'
 
+ENV["USE_CCACHE"] = Pod::Executable::which('ccache') ? "1" : "0"
```

Comment:
This is using [Pod::Executable::which](https://rubydoc.info/gems/cocoapods/Pod/Executable#which-class_method):
> Returns the absolute path to the binary with the given name on the current ‘PATH`, or `nil` if none is found.